packages/@webex/plugin-meetings/test/unit/spec/rtcMetrics/index.ts

```diff
@@ -4,10 +4,15 @@ import {assert} from '@webex/test-helper-chai';
 import sinon from 'sinon';
 import RTC_METRICS from '../../../../src/rtcMetrics/constants';
 
-const FAKE_METRICS_ITEM = {payload: ['fake-metrics']};
+const FAKE_METRICS_ITEM = {payload: ['{"type":"string","value":"fake-metrics","id":""}']};
+const FAILURE_METRICS_ITEM = {
+  name: "onconnectionstatechange",
+  payload: ['{"type":"string","value":"failed","id":""}'],
```

Comment:
why is id: blank ?

Reply:
This event does not have an id, most of the non stats-reports event don't have an id.